lib/topological_inventory/scheduler/database_tools.rb

```diff
@@ -0,0 +1,15 @@
+require 'acts_as_tenant'
+
+module TopologicalInventory
+  module Scheduler
+    class DatabaseTools
+      REFRESH_STATES_TIME_THRESHOLD = 1.week.freeze
+
+      def self.remove_old_records
+        threshold = DateTime.now - REFRESH_STATES_TIME_THRESHOLD
```

Comment:
We should add 2 logs there. Something like "Deleting older than..." and "Deleting older than..Completed"